docs/__main__.py

```diff
@@ -2,21 +2,24 @@
 
 import os
 import webbrowser
+
 try:
     from urllib.parse import urlunparse, quote
 except ImportError:
     from urlparse import urlunparse
     from urllib import quote
 
+
 def iterpath(path):
     path, last = os.path.split(path)
     if last:
         for p in iterpath(path):
             yield p
         yield last
 
+
 pkg_dir = os.path.dirname(os.path.abspath(__file__))
-main_page = os.path.join(pkg_dir, 'index.html')
+main_page = os.path.join(pkg_dir, 'generated/index.html')
```

Comment:
oops. Good catch.

Reply:
minor code nitpick here, `os.path.join(pkg_dir, 'generated/index.html')` can also be `os.path.join(pkg_dir, 'generated', 'index.html')`

Reply:
oh yeah you are right, definitively better to rely on the default directory separator provided by `os.path.join`, thanks!